apis/low_level_debug/src/tests.rs

```diff
@@ -0,0 +1,80 @@
+use super::*;
+use libtock_platform::ErrorCode;
+use libtock_unittest::{command_return, fake, ExpectedSyscall};
+
+#[test]
+fn driver_check() {
+    // Create a new fake kernel for the current thread, replacing avy previous one.
```

Comment:
I think the "Create a new fake kernel ... replacing ..." comments can be removed. Constructing a `fake::Kernel` will be an extremely common pattern in `libtock-rs`' tests, and `fake::Kernel::new`'s documentation already describes what it does.

Reply:
Done. I kept it initially since many comments stating the obvious exist in the source code.